src/Azure.Functions.Cli/Actions/LocalActions/InstallExtensionAction.cs

```diff
@@ -70,6 +72,14 @@ public override ICommandLineParserResult ParseArgs(string[] args)
 
         public async override Task RunAsync()
         {
+            var extensionBundleManager = ExtensionBundleHelper.GetExtensionBundleManager();
+            if (extensionBundleManager.IsExtensionBundleConfigured())
+            {
+                var hostFilePath = Path.Combine(Environment.CurrentDirectory, ScriptConstants.HostMetadataFileName);
+                ColoredConsole.WriteLine(WarningColor($"No action performed. Extension bundle is configured in {hostFilePath}"));
```

Comment:
I think few actions already call into the InstallExtensionsAction, like the start, and publish actions. Ideally it wouldn't display a warning for those commands.

Reply:
That is a good call, We need to prevent `func extension install` via those actions as well. May need to pass a flag to not show warnings if `func extension install` is run via that command